### DISCUSSION TITLE
2023 Zephyrus G16 GU603VV Support?

### DISCUSSION THREAD

**Discussion by ryanhossain9797** [seerge/g-helper#297] [Q&A] [answered]
Not seeing much talk about 2023 models. Is it supported?

**Comment by seerge** [comment, marked as answer]
It should :) you are fre to try.

**Reply by ryanhossain9797** [reply]
Way ahead of you
Seems to be working as far as I can see

I have two questions if you don't mind

1. Armoury crate had an option, "Modern standby helper" - Puts Laptop in hibernate after a certain amount of power drain in Modern Standby mode. Is there a counterpart to this in GHelper?

2. I remember Armoury Crate saying "running an external display requires the d-gpu". But here I'm plugging in an external display and it's running on the i-gpu just fine in Eco mode. So which is right? As far as I can tell the d-gpu really is off, GPU-Z can't detect it. Why does armoury crate say it can't be done. FYI the display is connected via a USB C dock, not the hdmi port.

Oh and of course, amazing work on this. I'm hopefully free of the Armoury Crate Bloat. Hope someday this'll work for the Ally.

**Reply by seerge** [reply]
1. I think this is handled by windows, not armoury / g-helper :) i use default "balanced" power plan from windows, and my laptop goes to hibernate in a couple of hours after being asleep. You can read about it setting here - https://www.tenforums.com/tutorials/108391-add-remove-hibernate-after-power-options-windows.html

What "modern standby helper" in armoury was doing, is just stopping couple of windows services to help laptop going to sleep. But if windows works ok - it shouldn't be needed 

2. External screen will work in Eco mode - if it's connected to iGPU directly (on G14 it's left usb-c port, i can "assume" on g16 - it's same?)

Does your G16 has animatrix? i think it's the only thing that won't work properly (for custom images), as number of leds is different, and i need to know exact sizes / numbers to adjust calculations  :)

**Reply by ryanhossain9797** [reply]
Nah G16 doesn't have Animatrix so can't help with testing, sorry.

It looks like for the 16 inch models Asus made the lower end ones G16 and the higher end ones M16. None of the G16 models have Anime Matrix.

On the g16 there are 2 usb c ports both on the left. I'm plugging into the one closer to the back. I'll try the other one and plugging into hdmi directly to see what happens.

Edit: looks like the back left usb c port works with igpu, the front left one and the hdmi port requires d-gpu

**Reply by ryanhossain9797** [reply]
> 1. I think this is handled by windows, not armoury / g-helper :) i use default "balanced" power plan from windows, and my laptop goes to hibernate in a couple of hours after being asleep. You can read about it setting here - https://www.tenforums.com/tutorials/108391-add-remove-hibernate-after-power-options-windows.html
> 
> What "modern standby helper" in armoury was doing, is just stopping couple of windows services to help laptop going to sleep. But if windows works ok - it shouldn't be needed 
> 
> 2. External screen will work in Eco mode - if it's connected to iGPU directly (on G14 it's left usb-c port, i can "assume" on g16 - it's same?)
> 
> Does your G16 has animatrix? i think it's the only thing that won't work properly (for custom images), as number of leds is different, and i need to know exact sizes / numbers to adjust calculations  :) 

So it does seem like the laptop won't go to hibernate after a while. Expecting windows to be ok is too much to ask I suppose.
